### PR TITLE
Navbar Component

### PR DESCRIPTION
**Pull request by Alee-star**
Navbar Component

### CODE REVIEW

File: src/components/Navbar/Navbar.css [hunk 1]
```diff
@@ -0,0 +1,79 @@
+.navpart {
```
Comment (vonnue-nidheesh) [inline src/components/Navbar/Navbar.css]: `navigation-part`

File: src/components/Navbar/Navbar.css [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  justify-content: center;
+}
+
+.nav-list {
+  display: flex;
+  padding: 0;
+  gap: 20px;
+  overflow-x: auto;
+  list-style-type: none;
+  align-items: center;
```
Comment (vonnue-nidheesh) [inline src/components/Navbar/Navbar.css]: Group styles.

for eg:
flex-related items are written together.

 ```
 display: flex;
  align-items: center;
  gap: 20px;
  padding: 0;
  overflow-x: auto;
  list-style-type: none;


```

File: src/components/Navbar/Navbar.css [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+}
+
+.list-item {
+  display: flex;
+  align-items: center;
+  height: 48px;
+  font-size: 16px;
+  border-radius: 30px;
+  color: #4a4a4a;
+  font-weight: 600;
```
Comment (vonnue-nidheesh) [inline src/components/Navbar/Navbar.css]: Same here font size , font-weight and color

File: src/components/Navbar/Navbar.css [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+
+.list-item.active {
+  background-color: black;
+  color: white;
+  border: 1px solid transparent;
+  padding: 0 20px;
+}
+
+.below-nav {
+  display: flex;
```
Comment (vonnue-nidheesh) [inline src/components/Navbar/Navbar.css:42]: Please group all class styles

File: src/components/Navbar/Navbar.jsx [hunk 5]
```diff
@@ -0,0 +1,42 @@
+import React, { useState } from "react";
+import Button from "../Button";
+import "./Navbar.css";
+
+const Navbar = () => {
+  const [activeTab, setActiveTab] = useState("Home");
+  const navItems = [
```
Comment (vonnue-nidheesh) [inline src/components/Navbar/Navbar.jsx]: Why are we using activeTab and navItems as variable names?

File: src/components/Navbar/Navbar.jsx [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+              key={item}
+              className={`list-item ${activeTab === item ? "active" : ""}`}
+              onClick={() => handleTabChange(item)}
+            >
+              {item}
+            </li>
+          ))}
+        </ul>
+      </div>
+      <div className="below-nav">
```
Comment (vonnue-nidheesh) [inline src/components/Navbar/Navbar.jsx]: `below-navigation`

File: src/components/Navbar/Navbar.jsx [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+        <h4>Free Stock Photos</h4>
+        <Button className="nav-button" label="Trending" isDisabled={true} />
```
Comment (vonnue-nidheesh) [inline src/components/Navbar/Navbar.jsx]: `<Button className="nav-button" label="Trending" isDisabled />`

If a prop is a boolean type just mention the variable name if we need the prop value as true.

File: src/components/Navbar/Navbar.jsx [hunk 8]
```diff
@@ -0,0 +1,42 @@
+import React, { useState } from "react";
+import Button from "../Button";
+import "./Navbar.css";
+
+const Navbar = () => {
+  const [navItem, setNavItem] = useState("Home");
```
Comment (vonnue-nidheesh) [inline src/components/Navbar/Navbar.jsx]: `selectedNavItem`

Comment (vonnue-nidheesh) [reply src/components/Navbar/Navbar.jsx]: https://codedamn.com/news/javascript/the-art-of-naming-variables-in-javascript-tips-and-best-practices

Comment (Alee-star) [reply src/components/Navbar/Navbar.jsx]: ok

**Review by vonnue-nidheesh — APPROVED** [review]

**Pull request merged**